kotest-extensions/src/jvmMain/kotlin/io/kotest/extensions/system/SystemExitExtensions.kt

```diff
@@ -80,7 +87,12 @@ class SystemExitException(val exitCode: Int) : RuntimeException()
 @Suppress("OverridingDeprecatedMember", "DEPRECATION")
 class NoExitSecurityManager(private val originalSecurityManager: SecurityManager?) : SecurityManager() {
 
-    override fun checkExit(status: Int) = throw SystemExitException(status)
+   var lastExitCode: Int = -1
```

Comment:
Perhaps changing this to be nullable instead of -1?

Reply:
Yes good call. Done.